spec/system/composer/prosemirror_editor_spec.rb

```diff
@@ -1140,4 +1140,148 @@ def body(title)
       )
     end
   end
+
+  describe "image toolbar" do
+    it "shows image toolbar when image is selected" do
+      cdp.allow_clipboard
+      open_composer_and_toggle_rich_editor
+      cdp.copy_test_image
+      cdp.paste
+
+      expect(rich).to have_css(".composer-image-node img")
+
+      rich.find(".composer-image-node img").click
+
+      expect(page).to have_css("[data-identifier='composer-image-toolbar']")
```

Comment:
Could be nice to have a `ComposerToolbar` page object that you can initialze for this kind of stuff.

```ruby
toolbar = ComposerToolbar.new("composer-image-toolbar")
expect(toolbar).to be_visible?
toolbar.click("zoom-out")
```